src/domains/ecommerce-app/api/ecommerce-transactions-service/v1/_openapi.json.tpl

```diff
@@ -816,6 +836,24 @@
             "properties": {
               "feeTotal": {
                 "$ref": "#/components/schemas/AmountEuroCents"
+              },
+              "paymentGateway": {
```

Comment:
is this an old mismatch?

Reply:
eCommerce transaction service api where misaligned respect to [service api's](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/b2c7692bfb60c1cff78ce28a24cc1136a675f30f/api-spec/v1/transactions-api.yaml#L678-L700) so, during this api update I align the api specs with the transactions-service one's